### PR TITLE
feat: added the order-API

### PR DESCRIPTION
**Pull request by Kavindu-248**

### CODE REVIEW

File: apps/dispensary/views.py [hunk 1]
```diff
@@ -20,31 +21,48 @@ def retrieve(self, request, pk=None):
...
+                Order.objects.filter(form_assesment=prescription.form_assesment).update(prescription=prescription)
+                serializer.save()
+        elif prescription.prescription_status == PrescriptionStatus.REJECTED:
+            raise ValidationError('This prescription is already canceled hence you cannot update it')
+        else:
+            super().perform_update(serializer)
+
+class PharmacyViewSet(viewsets.ViewSet):
+
     def list(self, request):
```
Comment (Raam124) [inline apps/dispensary/views.py:45]: why are you using list here ?

File: apps/dispensary/views.py [hunk 2]
```diff
@@ -20,31 +21,48 @@ def retrieve(self, request, pk=None):
...
-        queryset = Prescription.objects.all()
-        serializer = PrescriptionSerializer(queryset, many=True)
+        queryset = Pharmacy.objects.all()
+        serializer = PharmacySerializer(queryset, many=True)
         return Response(serializer.data)
 
     def update(self, request, pk=None):
```
Comment (Raam124) [inline apps/dispensary/views.py:50]: what's the use of this update method ?

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/dispensary/views.py (hunk 1 above)
Comment (Kavindu-248) [reply apps/dispensary/views.py:45]: @Raam124  prior I made pharmacy as a ModelViewset, and then we discussed we don't need all the CRUD operations for pharmacy and changed it  just to read or update the data.

File: apps/dispensary/views.py (hunk 2 above)
Comment (Kavindu-248) [reply apps/dispensary/views.py:50]: @Raam124  prior I made pharmacy as a ModelViewset, and then we discussed we dont need all the CRUD operations for pharmacy and changed it  just to read or update the data.

File: apps/dispensary/views.py (hunk 1 above)
Comment (Raam124) [reply apps/dispensary/views.py:45]: so why do you have the pharmacy API inside order apis PR ?

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/dispensary/views.py (hunk 1 above)
Comment (Kavindu-248) [reply apps/dispensary/views.py:45]: @Raam124 , This was the last synced code from the dev branch, in our old Order PR we were discussing some changes before we changed the models, after the Model change, and with this new PR I made sure to include those necessary changes.

Comment (Raam124) [reply apps/dispensary/views.py:45]: well remove those changes from this PR and make it a separate PR then

**Review by Raam124 — CHANGES_REQUESTED** [review]

**Review by Raam124 — APPROVED** [review]

**Pull request merged**